docs/docs/deployments.md

```diff
@@ -18,6 +18,99 @@ Multiple deployments can be performed at the same, as long as one application is
 If a deployment is requested, that tries to change an application that is already being changed by another active deployment, 
 the new deployment request will be rejected.
 
+## Basics
+
+Let us start with a simple example of a deployment: an app that prints `Hello Marathon` to stdout and then sleeps for 5 sec, in an endless loop.
+You would use the following application resource (in JSON format) to describe the deployment: 
+
+```json
+{
+    "id": "basic-0", 
+    "cmd": "while [ true ] ; do echo 'Hello Marathon' ; sleep 5 ; done",
+    "cpus": 0.1,
+    "mem": 10.0,
+    "instances": 1
+}
+```
+
+Note that `cmd` in the above example is the command that gets executed. Its value is wrapped by the underlying Mesos executor via `/bin/sh -c ${cmd}`.
+
+<p class="text-center">
+  <img src="{{ site.baseurl}}/img/marathon-basic-0.png" width="800" height="573" alt="Marathon deployment example: simple bash command">
+</p>
+
+What happens here is that Marathon hands over execution to Mesos. Mesos executes each task in its own sandbox environment.
+The sandbox is a special directory on each slave that acts as the execution environment (from a storage perspective) and also contains relevant log files as well as `stderr` and `stdout` for the command being executed. See also the role of the sandbox in [debugging distributed apps](https://docs.mesosphere.com/tutorials/debugging-a-mesosphere-cluster/).
+
+For any non-trivial deployment you typically depend on a collection of resources, that is, files and/or archives of files. To deal with this, Marathon has the concept of `uris`. It leverages the Mesos fetcher to do the legwork in terms of downloading (and potentially) extracting resources.
+
+But before we dive into this topic, let's have a look at an example:
+
+```json
+{
+    "id": "basic-1", 
+    "cmd": "cool-script.sh",
```

Comment:
Resolved with adding a note explaining the difference between pre and post Mesos 0.21